snapcraft/models/project.py

```diff
@@ -426,6 +445,54 @@ def _validate_default_provider(cls, default_provider):
         return default_provider
 
 
+class Platform(pydantic.BaseModel):
```

Comment:
This can probably inherit from `CraftBaseModel` instead, and then you can drop the Config

Reply:
Fixed.